src/setup.js

```diff
@@ -25,11 +26,27 @@ async function installHelmPlugins(plugins) {
 }
 
 async function installHelmfile(version) {
+  if (semvercompare(version.replace(/^v/,), "0.145.0") >= 0) {
```

Comment:
The second argument is required. This shoud work.

`if (semvercompare(version.replace(/^v/, ''), "0.145.0") >= 0) {`